src/holofuel/components/SideMenu/SideMenu.js

```diff
@@ -63,11 +67,19 @@ export default function SideMenu ({
               Profile
             </Link>
           </li>
-          {process.env.REACT_APP_HOLOFUEL_APP !== 'true' && <li>
-            <Link to='/admin/' styleName='nav-link'>
+          {process.env.REACT_APP_HOLOFUEL_APP !== 'true' && <li styleName='last-list-item'>
+            <Link to='/admin/' styleName='nav-link last-nav-link'>
```

Comment:
can you do this using [:last-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child) instead of a separate style?